packages/core/src/components/forms/_control-group.scss

```diff
@@ -46,6 +46,8 @@ Styleguide pt-control-group
 */
 
 .pt-control-group {
+  // create a new stacking context to isolate all the z-indices
+  @include force-hardware-acceleration();
```

Comment:
this is strange to read. would be better to make a new mixin called `new-stacking-context()`, even if all it does is delegate to this one